tests/cases/conformance/types/thisType/thisTypeInFunctions2.ts

```diff
@@ -0,0 +1,26 @@
+interface Arguments {
```

Comment:
can you add a test that does not have this, something with `[propName : string]:any`